src/main/java/uk/gov/hmcts/reform/et/syaapi/service/CaseDocumentService.java

```diff
@@ -0,0 +1,214 @@
+package uk.gov.hmcts.reform.et.syaapi.service;
+
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.tika.Tika;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.et.syaapi.models.CaseDocument;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static uk.gov.hmcts.reform.ccd.client.model.Classification.PUBLIC;
+
+/**
+ * CaseDocumentService provides access to the document upload service API, used to upload documents that are
+ * associated to a specific case record held in CCD.
+ * <p/>
+ * This relies upon the following configurations to be set at an environment level:
+ * <ul>
+ *     <li>CASE_DOCUMENT_AM_URL</li>
+ *     <li>CASE_DOCUMENT_AM_MAX_RETRIES</li>
+ * </ul>
+ */
+@Slf4j
+@Service
+public class CaseDocumentService {
+
+    private static final String JURISDICTION = "EMPLOYMENT";
```

Comment:
This is already defined here 
https://github.com/hmcts/et-sya-api/blob/0b0a7f1efa1f286290304d152bc17ddcc42c4e47/src/main/java/uk/gov/hmcts/reform/et/syaapi/constants/EtSyaConstants.java#L8